jenkins/deploy.sh

```diff
@@ -82,15 +83,15 @@ $DEPLOY_CMD -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \
 # Distribution jar is a shaded artifact so use the reduced dependency pom.
 $DEPLOY_CMD -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \
             $SRC_DOC_JARS \
-            -Dfile=$FPATH.jar -DgroupId=com.nvidia -DartifactId=$ART_ID -Dversion=$ART_VER -DpomFile=./dist/target/dependency-reduced-pom.xml
```

Comment:
We removed the `depednecy-reduce` script from dist pom.xml from branch-21.12, so not such `./dist/target/dependency-reduced-pom.xml` will be generated.